### ISSUE TITLE
Unable to create SttpClient for ZIO synchronous backend

### ISSUE THREAD

**Issue by steinybot** [softwaremill/sttp#637]
I'm just trying to sttp for the first time using the ZIO synchronous backend.

I was expecting this to work but it doesn't:
```
  private def foo = {
    val request = basicRequest.post(Uri("http://example.com")).body("hello")
    SttpClient.send(request).provideLayer(HttpClientZioBackend.layer())
  }
```

The problem is that `HttpClientZioBackend.layer()` returns a `ZLayer[Blocking, Throwable, Has[SttpBackend[BlockingTask, ZStream[Blocking, Throwable, Byte], WebSocketHandler]]]` but `send` requires a `Has[SttpBackend[BlockingTask, ZStream[Blocking, Throwable, Byte], NothingT]]`.

It looks like this is a regression introduced in https://github.com/softwaremill/sttp/pull/603

**Comment by adamw** [comment]
Yes you're right, we didn't update the `SttpClient` type alias. Will fix shortly :) Thanks!

**Comment by adamw** [comment]
The fully-compiling example will be:

```
private def foo = {
    import sttp.client._
    import zio.blocking._
    import sttp.client.httpclient.zio._
    val request = basicRequest.post(uri"http://example.com").body("hello")
    SttpClient.send(request).provideSomeLayer(HttpClientZioBackend.layer()).provideLayer(Blocking.live)
  }
```

(we need to use `provideSome` as two layers are needed - one for the client, one for blocking)

**Comment by adamw** [comment]
Fixed in 2.2.2
